app/models/katello/ping.rb

```diff
@@ -195,10 +195,7 @@ def pulp3_without_auth(url)
         end
 
         workers = json["online_workers"] || []
-
-        unless workers.count > 1
-          fail _("Not all necessary pulp workers running at %s.") % url
-        end
+        fail _("Not all necessary pulp workers running at %s.") % url if workers.empty?
```

Comment:
Could the error message be "No pulp workers running..." if it's failing when workers is empty?

Reply:
updated!